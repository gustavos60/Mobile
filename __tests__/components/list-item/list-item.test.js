import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import ListItem from '../../../src/components/list-item';

const mockedProps = {
  place: {
    latitude: 11,
    longitude: 22,
    name: 'Gustavo',
    uri: 'https://dubsism.files.wordpress.com/2017/12/image-not-found.png?w=547',
  },
};

describe('ListItem tests', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<ListItem {...mockedProps} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
