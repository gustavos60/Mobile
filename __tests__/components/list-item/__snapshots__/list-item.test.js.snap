// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem tests renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "height": 100,
      "marginBottom": 10,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://dubsism.files.wordpress.com/2017/12/image-not-found.png?w=547",
      }
    }
    style={
      Object {
        "borderRadius": 10,
        "height": "90%",
        "marginRight": 10,
        "width": "30%",
      }
    }
  />
  <View>
    <Text
      style={
        Object {
          "fontSize": 18,
          "marginBottom": 5,
          "overflow": "visible",
        }
      }
    >
      Gustavo
    </Text>
    <Text
      style={
        Object {
          "marginBottom": 5,
        }
      }
    >
      Latitude:
       
      11
    </Text>
    <Text
      style={
        Object {
          "marginBottom": 5,
        }
      }
    >
      Longitude:
       
      22
    </Text>
  </View>
</View>
`;
